src/theme/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { Dimensions } from "react-native";
+
+export const screenWidth = Dimensions.get('window').width;
+export const screenHeight = Dimensions.get('window').height;
+
+export const colors = {
```

Comment:
Suggestion: 
You can create separate files for colors, images, and dimensions as well.